### ISSUE TITLE
Ignore

### ISSUE THREAD

**Issue by gundalow** [ansible/ansible-lint#759]
##### Issue Type

- Bug Report

##### Ansible and Ansible Lint details

```console (paste below)
ansible --version

ansible-lint --version

```

- ansible installation method: one of source, pip, OS package
- ansible-lint installation method: one of source, pip, OS package

##### STEPS TO REPRODUCE

```console (paste below)

```

##### Desired Behaviour

Possible security bugs should be reported via email to `security@ansible.com`

##### Actual Behaviour

Please give some details of what is actually happening.
Include a [minimum complete verifiable example] with:
- playbook
- output of running ansible-lint
- if you're getting a stack trace, output of
  `ansible-playbook --syntax-check playbook`

```paste below

```

[minimum complete verifiable example]: http://stackoverflow.com/help/mcve
